setup.py

```diff
@@ -40,7 +40,7 @@
         ],
         'tensorflow-gpu': [
             'tensorflow-gpu>=1.10.0,<2.0',
-            'tensorflow-probability-gpu>=0.3.0'
+            'tensorflow-probability>=0.3.0'
```

Comment:
Is `tensorflow-probability` itself gpu version?

Reply:
There is no longer a separate GPU-specific build of `tensorflow-probability` after version 0.5.0. Users can select the version of `tensorflow` they wish to use (CPU or GPU), and `tensorflow-probability` will work with both.